packages/components/organisms/f-cookie-banner/test/visual/f-cookie-banner-legacy.visual.spec.js

```diff
@@ -0,0 +1,29 @@
+import forEach from 'mocha-each';
+
+const CookieBannerLegacy = require('../../test-utils/component-objects/f-cookie-banner-legacy.component');
+
+let cookieBanner;
+
+forEach([
+    'desktop',
+    'mobile'
+])
+.describe('Legacy - f-cookie-banner - %s - Visual Tests', device => {
+    beforeEach(async () => {
+        // Arrange
+        if (device === 'mobile') {
+            browser.setWindowSize(414, 731);
```

Comment:
Why do we set a specific window size for "mobile" but not other screen sizes i.e tablet etc? Should we also refer to these sizes as `narrow, mid` to match fozzie variables? (I see it's used throughout the repo tho)